src/libs/workspace-utils.test.ts

```diff
@@ -1,24 +1,24 @@
 import { defaultAzureWorkspace, defaultGoogleWorkspace } from 'src/analysis/_testData/testData';
 
-import { isValidWsExportTarget } from './workspace-utils';
+import { isValidWsExportTarget, WorkspaceWrapper } from './workspace-utils';
 
 describe('isValidWsExportTarget', () => {
   it('Returns true because source and dest workspaces are the same', () => {
     // Arrange
-    const sourceWs = {
+    const sourceWs: WorkspaceWrapper = {
       ...defaultGoogleWorkspace,
       workspace: {
-        ...defaultGoogleWorkspace,
-        authorizationDomain: [{}],
+        ...defaultGoogleWorkspace.workspace,
```

Comment:
Adding the : WorkspaceWrapper type safety to the sourceWs declaration here is HIGHLY recommended to catch bugs like this.... missing the .workspace sub-property expectation is pretty bad...